src/sentry/relay/config/__init__.py

```diff
@@ -328,7 +338,7 @@ def to_dict(self):
             for (key, value) in data.items()
         }
 
-    def to_json_string(self):
+    def to_json_string(self) -> Any:
```

Comment:
Can we make this `-> str`?

Reply:
If i change it to:
```
    def to_json_string(self) -> str:
        """
        >>> x = _ConfigBase( a = _ConfigBase(b = _ConfigBase( w=[1,2,3])))
        >>> x.to_json_string()
        '{"a": {"b": {"w": [1, 2, 3]}}}'
        :return:
        """
        data = self.to_dict()
        return utils.json.dumps(data)
```
src/sentry/relay/config/__init__.py:346: error: Returning Any from function declared to return "str"  [no-any-return]
Found 1 error in 1 file (checked 575 source files)

can't explain it, also i doubled checked `utils.json.dumps` returns `str`

Reply:
I would `# type: ignore` the line then, this is clearly wrong

Reply:
yeah i realised it so i added as @untitaker suggested `# type: ignore` and it worked.

Reply:
it would be better to add that file to mypy if possible

Reply:
@asottile-sentry in this PR or in seperate?
```
src/sentry/utils/json.py:19: error: Library stubs not installed for "simplejson" (or incompatible with Python 3.8)  [import]
src/sentry/utils/json.py:19: note: Hint: "python3 -m pip install types-simplejson"
src/sentry/utils/json.py:19: note: (or run "mypy --install-types" to install all missing stub packages)
src/sentry/utils/json.py:19: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
src/sentry/utils/json.py:24: error: Function is missing a type annotation  [no-untyped-def]
src/sentry/utils/json.py:54: error: Class cannot subclass "JSONEncoder" (has type "Any")  [misc]
src/sentry/utils/json.py:57: error: Function is missing a type annotation  [no-untyped-def]
src/sentry/utils/json.py:60: error: Call to untyped function "iterencode" in typed context  [no-untyped-call]
src/sentry/utils/json.py:63: error: Function is missing a type annotation  [no-untyped-def]
src/sentry/utils/json.py:93: error: Function is missing a return type annotation  [no-untyped-def]
src/sentry/utils/json.py:93: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]
src/sentry/utils/json.py:98: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]
src/sentry/utils/json.py:101: error: Returning Any from function declared to return "str"  [no-any-return]
src/sentry/utils/json.py:101: error: Call to untyped function "encode" in typed context  [no-untyped-call]
src/sentry/utils/json.py:102: error: Returning Any from function declared to return "str"  [no-any-return]
src/sentry/utils/json.py:105: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]
src/sentry/utils/json.py:109: error: Function is missing a type annotation for one or more arguments  [no-untyped-def]
src/sentry/utils/json.py:117: error: Function is missing a type annotation  [no-untyped-def]
src/sentry/utils/json.py:118: error: Call to untyped function "encode" in typed context  [no-untyped-call]
src/sentry/utils/json.py:121: error: Function is missing a type annotation  [no-untyped-def]
```

Reply:
bleh, I can do it in a separate PR -- thought it might be easier than this :S

Reply:
thanks!